hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/processors/StreamToStreamJoinPInnerTest.java

```diff
@@ -419,7 +420,24 @@ static Expression<Boolean> createConditionFromPostponeTimeMap(
             wmKeyToColumnIndexMap.put((byte) wmKeyToColumnIndex[i], (byte) wmKeyToColumnIndex[i + 1]);
         }
 
+        // specific case for simple equi-join : l.time == r.time
+        if (postponeTimeMap.size() == 2
+                && postponeTimeMap.get((byte) 0).get((byte) 1) == 0L
```

Comment:
This will fail with NPE if there are no entries with key 0 or 1.

Reply:
Doesn't matter for test scenarios for processor with simple `l.time == r.time` condition